### ISSUE TITLE
SQL Error when starting service

### ISSUE THREAD

**Issue by mmosteller** [crowbarmaster/BedrockService#1]
I found your modified version of the service after using the original for a while. I haven't had any issues with the original except for the desire to run the client remotely instead of popping open a RD window. However, I followed your instructions after downloading the most recent build and get this message when I run start it.

```ERROR [1] (:0) - The service failed to start.
System.ServiceProcess.TimeoutException: Time out has expired and the operation has not been completed.
   at System.ServiceProcess.ServiceController.WaitForStatus(ServiceControllerStatus desiredStatus, TimeSpan timeout)
   at Topshelf.Runtime.Windows.WindowsHostEnvironment.StartService(String serviceName, TimeSpan startTimeOut)
   at Topshelf.Hosts.StartHost.Run()
```

The service install goes fine and if I try to run the application without creating the service it spits out

```Error Starting BedrockServiceWrapper The given key was not present in the dictionary.
ERROR [1] (:0) - An exception occurred
Topshelf.TopshelfException: The service failed to start (return false).
   at Topshelf.Hosts.ConsoleRunHost.Run()
```

So basically the same thing just without the references to it being a service.

**Comment by ClasSicKilian** [comment]
Same Error, any updates?

```
ERROR [1] (:0) - The service failed to start.
System.ServiceProcess.TimeoutException: Time out has expired and the operation has not been completed.
   at System.ServiceProcess.ServiceController.WaitForStatus(ServiceControllerStatus desiredStatus, TimeSpan timeout)
   at Topshelf.Runtime.Windows.WindowsHostEnvironment.StartService(String serviceName, TimeSpan startTimeOut)
   at Topshelf.Hosts.StartHost.Run()
```

**Comment by crowbarmaster** [comment]
Hello!! My applogies. I did not see this issue posted. If your still looking for an answer, I'd be happy to help. For starters, can you recall for sure that you set "BedrockService.exe" to run as admin under compatibility settings? The batch files will not pass elevated privileges to the executable. A right-click > Run as Administrator on BedrockService.exe should fix this as well. Let me know!
